lib/pantry.rb

```diff
@@ -0,0 +1,30 @@
+class Pantry
+
+    attr_reader :stock
+
+    def initialize
+        @stock = Hash.new(0)
+    end
+    
+    def restock(ingredient, qty)
+        @stock[ingredient] += qty
+    end
+
+    def stock_check(ingredient = "")
+        if ingredient == ""
```

Comment:
I am not sure where this empty string is coming from, but think about your hash with it's default value. What we want this method to do is to just spit out the value of the given ingredient. The ingredients are the keys in your hash. So all we needed in this method was `@stock[ingredient]` and that would spit back either the value, or the default value.